modules/signing_scriptworker/templates/dep-passwords-mobile.json.erb

```diff
@@ -0,0 +1,5 @@
+{
+    "project:mobile:reference-browser:releng:signing:cert:dep-signing": [
```

Comment:
This LGTM. If this lands now, it will be fine, but won't affect any existing instance as we don't currently have a dep mobil signing instance under `moco-nodes` configured with an `env_config` to use the `dep-passwords-mobile.erb` template file. We only have production instances right now, that look like [this](https://github.com/mozilla-releng/build-puppet/blob/master/manifests/moco-nodes.pp#L298).

Reply:
I'd remove this file if we don't go with the staging environment. Otherwise, we should add `env_config`. Leaving this file alone may confuse our future selves